### PR TITLE
update dg

### PR DESCRIPTION
**Pull request by mollyyoung**
more to come

### CODE REVIEW

File: source/guides/dg.md [hunk 1]
```diff
@@ -1 +1,16 @@
 # Demand Generation 
+## SDR Playbook
```
Comment (mollyyoung) [inline source/guides/dg.md]:
```suggestion
```

File: source/guides/dg.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+### Lead Flow
```
Comment (mollyyoung) [inline source/guides/dg.md]:
```suggestion
## Lead Flow
```

File: source/guides/dg.md [hunk 3]
```diff
@@ -1 +1,16 @@
...
+Detailed territory assignment sheet located [here](https://docs.google.com/spreadsheets/d/1RcO7ayG6x1aDVxD3AfvSfjlzvzJkvU2GB0wvNIEDhu8/edit)(internal doc).
+More detail to come on rules of engagement. 
+
+#### Lead Status
```
Comment (mollyyoung) [inline source/guides/dg.md]:
```suggestion
### Lead Status
```

**Comment by mollyyoung** [comment]
Updated layout and titling

File: source/guides/dg.md [hunk 4]
```diff
@@ -1 +1,15 @@
...
+Detailed territory assignment sheet located [here](https://docs.google.com/spreadsheets/d/1RcO7ayG6x1aDVxD3AfvSfjlzvzJkvU2GB0wvNIEDhu8/edit)(internal doc).
+More detail to come on rules of engagement. 
+
+### Lead Status
+When a lead comes into Salesforce, it progresses through the lead statuses defined below:
+* **New** New new lead record, not enough activity to be considered an MQL.
+* **MQL** A Marketing Qualified Lead (MQL) is a lead that has reached a lead scoring threshold (100 points). Lead scoring model is based on firmographic, demographic, and behavioral information and can be seen [here](https://docs.google.com/spreadsheets/d/1ZMF7ppWS_Naf-kIFgRymLWwHRVkB09VTqB7wv-AXOnM/edit#gid=0) (internal doc).
+* **Accepted** Lead owner has accepted the lead and will work to qualify. *This will remove from marketing nurture.*
+* **Qualifying** Lead owner is engaged in 2-way conversation with the lead.
+* **Qualified** Lead owner has uncovered and recorded in Salesforce the required sales qualified lead (SQL) criteria. Lead has committed to a defined next step.
```
Comment (mollyyoung) [inline source/guides/dg.md]:
```suggestion
* **Qualified** Lead owner has uncovered and recorded in Salesforce the required sales accepted (SAO) or sales qualified lead (SQL) criteria. Lead has committed to a defined next step. 
```

**Comment by it33** [comment]
Thanks @mollyyoung let us know when ready to merge?

**Comment by it33** [comment]
Ah, looks like assigned to me, merging,

**Comment by mollyyoung** [comment]
Thank you!

**Pull request merged**